src/bidiMapper/domains/network/NetworkRequest.ts

```diff
@@ -137,13 +143,15 @@ export class NetworkRequest {
     return Boolean(this.#request.info);
   }
 
-  #isBlockedInPhase(phase?: Network.InterceptPhase) {
-    return this.#networkStorage.requestBlockedBy(this, phase).size > 0;
+  #isBlockedByInPhase(phase: Network.InterceptPhase) {
```

Comment:
Is blocked is confusing I think? Let's check if there are interceptors for the phase instead so renaming `isInterceptedInPhase`